releasenotes/notes/remove-span-get-set-meta-c6fb2528d198414d.yaml

```
@@ -0,0 +1,16 @@
+---
+deprecations:
```

This should be corrected

```suggestion
upgrade:
```